dxw/dxw-cache-control
# Add the ability to control the cache on specific pages

This PR adds the ability for the cache to be controlled
on specific pages in a site using this plugin, allowing any post
to have its own specific cache setting.

To test:
on a local instance which has the plugin installed and enabled
1. confirm that you can now configure individual posts.
2. with development mode enabled load an archive page or 404 page and confirm that this doesn't produce any unexpected errors and that the meta-cc-post-id header returns the value 0
3. load the configured post and confirm that the header value returned is the expected value
4. load any non-configured post and confirm that the header value returned is the expected value 

for items 3 and 4 the dev headers Meta-Cc-Final-Configured-Max-Age and 'Meta-cc-individual-page-cache-setting-triggered can help confirm this, especially if the local instance doesn't allow browsing while not logged in.
